pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -642,6 +643,122 @@ func TestNewFrameworkPluginDefaults(t *testing.T) {
 	}
 }
 
+// fakeNoopPlugin doesn't implement interface framework.EnqueueExtensions.
+type fakeNoopPlugin struct{}
+
+func (pl *fakeNoopPlugin) Name() string { return "fakeNoop" }
+
+type fakeNodePlugin struct{}
+
+func (pl *fakeNodePlugin) Name() string { return "fakeNode" }
+
+func (pl *fakeNodePlugin) EventsToRegister() *framework.ClusterEventSet {
+	return &framework.ClusterEventSet{
+		NonInterested: []framework.ClusterEvent{
+			{Resource: framework.Node, ActionType: framework.Delete},
+			{Resource: framework.CSINode, ActionType: framework.Update | framework.Delete},
+		},
+	}
+}
+
+type fakePodPlugin struct{}
+
+func (pl *fakePodPlugin) Name() string { return "fakePod" }
+
+func (pl *fakePodPlugin) EventsToRegister() *framework.ClusterEventSet {
+	return &framework.ClusterEventSet{
+		Interested: []framework.ClusterEvent{
+			{Resource: framework.Pod, ActionType: framework.All},
+			{Resource: framework.Node, ActionType: framework.Add | framework.Update},
+			{Resource: framework.Service, ActionType: framework.Delete},
+		},
+		NonInterested: []framework.ClusterEvent{framework.WildCardEvent},
+	}
+}
+
+// fakeNoopPlugin implement interface framework.EnqueueExtensions, but returns nil
```

Comment:
use "fakeNoopRuntimePlugin" instead of "fakeNoopPlugin" to distinguish this one from the plugin you defined above.

Reply:
Thanks for the catch. Updated.